yew/src/agent/worker/private.rs

```diff
@@ -31,13 +38,30 @@ where
     type Agent = AGN;
 
     fn spawn_or_join(callback: Option<Callback<AGN::Output>>) -> Box<dyn Bridge<AGN>> {
+        let id = PRIVATE_ID_COUNTER.fetch_add(1, Ordering::Relaxed);
         let callback = callback.expect("Callback required for Private agents");
         let handler = move |data: Vec<u8>,
                             #[cfg(feature = "std_web")] worker: Value,
                             #[cfg(feature = "web_sys")] worker: &Worker| {
             let msg = FromWorker::<AGN::Output>::unpack(&data);
             match msg {
                 FromWorker::WorkerLoaded => {
+                    QUEUE.with(|queue| {
+                        queue.insert_loaded_agent(id);
+
+                        let mut msg_queue = queue.borrow_msg_queue_mut();
+                        if let Some(msgs) = msg_queue.get_mut(&id) {
+                            for msg in msgs.drain(..) {
+                                cfg_match! {
+                                    feature = "std_web" => ({
+                                        let worker = &worker;
+                                        js! {@{worker}.postMessage(@{msg});};
+                                    }),
+                                    feature = "web_sys" => worker.post_message_vec(msg),
```

Comment:
Can we use the `send_to_remote` method here?

Reply:
Currently `send_to_remote` takes a  `ToWorker` type, while the queue holds messages in a `vec<vec<u8>>` format

Reply:
Got it, that's fine then